### DISCUSSION TITLE
Group with user and nested groups

### DISCUSSION THREAD

**Discussion by sla82** [dalibo/ldap2pg#680] [Q&A]
Hello everyone,

I would like to create a historically grown authorization group from the AD. Unfortunately, this contains users and other groups. What is the best way to synchronize them with ldap2pg?

PSQL-Owner (Permission Group, with included users and another groups, which need permission)
psql.admin.user
psql.backup.user
psql.admin.group
psql.it.group

I have also tried with the filter (objectClass=group) and (objectClass=User), but groups and users are always created both

Many thanks in advance

**Comment by bersace** [comment]
Hi @sla83, did you tried `(memberOf=)` filter clause ?
